wp-content/themes/amplify/amplify-feed.php

```diff
@@ -4,6 +4,33 @@
  * Template Name: Amplify Feed Template
  * Description: Shows the saved links feed that can be embedded on external sites.
  */
+
+if( $_GET['horizontal'] && 'true' == $_GET['horizontal'] ){
+    $body_class = 'horizontal';
+} else {
+    $body_class = 'vertical';
+}
+
+if( $_GET['show_link_descriptions'] && 'true' == $_GET['show_link_descriptions'] ){
+    $saved_links_class = 'show-link-description';
+} else {
+    $saved_links_class = '';
+}
+
+if( $_GET['hide_header'] && 'true' == $_GET['hide_header'] ) {
+    $hide_header = 'hide-header';
+} else {
+    $hide_header = '';
+}
+
+if( $_GET['rows'] && 2 == $_GET['rows'] ){
+    $rows = 'rows-2';
+} else if( $_GET['rows'] && 3 == $_GET['rows'] ){
+    $rows = 'rows-3';
+} else {
+    $rows = 'rows-1';
```

Comment:
with `&horizontal=true&rows=1`, on wide viewports, there appear three columns:

<img width="1032" alt="Screen Shot 2019-12-16 at 11 27 38" src="https://user-images.githubusercontent.com/1754187/70924168-13ee7800-1ff7-11ea-9fd7-691ababcd207.png">

Reply:
@benlk https://github.com/INN/umbrella-inndev/pull/131/commits/48107a826345437883866229e71ff5f828ec6916 should fix that.

Reply:
Fixed. On the horizontal layout, do we want to make sure the logo is aligned to the top of the widget?

Reply:
@benlk I'm not sure. When I showed this to @kaylima and @MirandaEcho they didn't specifically say anything about alignment for it.

Reply:
Then I guess it'll come up in feedback from the user testing.